cloudconfig/instancecfg/instancecfg.go

```diff
@@ -633,13 +633,9 @@ func (cfg *ControllerConfig) VerifyConfig() error {
 	return nil
 }
 
-// DefaultBridgePrefix is the prefix for all network bridge device
-// name used for LXC and KVM containers.
-const DefaultBridgePrefix = "br-"
-
 // DefaultBridgeName is the network bridge device name used for LXC and KVM
 // containers
-const DefaultBridgeName = DefaultBridgePrefix + "eth0"
+const DefaultBridgeName = "br-eth0"
```

Comment:
provider/maas/environ.go:1359, in a log message. I'm more than happy to remove it but should I really replace it with generic br-eth0?